custom-recipes/meaningcloud-deep-categorization/recipe.json

```
@@ -1,119 +1,157 @@
+                },
+                {
+                    "value": "es",
+                    "label": "Spanish"
+                }
             ],
             "defaultValue": "en"
         },
+        {
+            "name": "separator_configuration",
+            "label": "Configuration",
+            "type": "SEPARATOR"
+        },
         {
             "name": "model",
-            "label": "Model",
-            "description": "Model to use in the classification.",
+            "label": "Predefined model",
             "type": "SELECT",
-            "visibilityCondition" : "model.predefined_model == true",
             "selectChoices": [
-                {"value": "Emotion", "label": "Emotion Recognition"},
-                {"value": "IAB_2.0", "label": "IAB 2.0"},
-                {"value": "IntentionAnalysis", "label": "Intention Analysis"},                
-                {"value": "VoC-Banking", "label": "VoC - Banking"},
-                {"value": "VoC-Generic", "label": "VoC - Generic"},
-                {"value": "VoC-Insurance", "label": "VoC - Insurance"},
-                {"value": "VoC-Retail", "label": "VoC - Retail"},
-                {"value": "VoC-Telco", "label": "VoC - Telco"},
-                {"value": "VoE-ExitInterview", "label": "VoE - Exit Interview"},
-                {"value": "VoE-Organization", "label": "VoE - Organization"},
-                {"value": "VoE-Performance", "label": "VoE - Performance"}
-            ]
+                {
+                    "value": "Emotion",
+                    "label": "Emotion Recognition"
+                },
+                {
+                    "value": "IAB_2.0",
+                    "label": "IAB 2.0"
+                },
+                {
+                    "value": "IntentionAnalysis",
+                    "label": "Intention Analysis"
+                },
+                {
+                    "value": "VoC-Banking",
+                    "label": "VoC - Banking"
+                },
+                {
+                    "value": "VoC-Generic",
+                    "label": "VoC - Generic"
+                },
+                {
+                    "value": "VoC-Insurance",
+                    "label": "VoC - Insurance"
+                },
+                {
+                    "value": "VoC-Retail",
+                    "label": "VoC - Retail"
+                },
+                {
+                    "value": "VoC-Telco",
+                    "label": "VoC - Telco"
+                },
+                {
+                    "value": "VoE-ExitInterview",
+                    "label": "VoE - Exit Interview"
+                },
+                {
+                    "value": "VoE-Organization",
+                    "label": "VoE - Organization"
+                },
+                {
+                    "value": "VoE-Performance",
+                    "label": "VoE - Performance"
+                }
+            ],
+            "defaultValue": "IntentionAnalysis"
```

```suggestion
            "defaultValue": "IAB_2.0"
```
The IAB 2.0 model is a better option for the default value, as it'is available for every user from the start. The Intention Analysis model is part of a vertical pack, so you need to request access first or you will get an error.

---

Good point. From a novice user perspective, I would suggest amending the labels for model selectChoices to:
- mark which models are part of vertical packs
- Extend acronyms (for instance, I am not sure everyone knows what VoE and IAB means) 
What do you think?